### PR TITLE
pushing local redisssl docs

### PR DESCRIPTION
**Pull request by SethSanchez**

### CODE REVIEW

File: source/redis_ssl_faq.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+What do I need to do with my application to use SSL Encryption
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You will need to use some type of SSL proxy in order to handle the SSL “handshake” that takes place between your client and the ObjectRocket Redis endpoint.  An example of this would be stunnel.  Once you have this in place, all that you need to do is point your application to the desired SSL encryption connection string.  There should be no additional code changes required on your end.  You may chose to do some additional verification of the server that you are connecting to via certificate “fingerprint” confirmation (see certificates and fingerprints below).
+
+Certificates and Fingerprints
```
Comment (dwjohn) [inline source/redis_ssl_faq.rst]: Get rid of "and Fingerprints" out of the title

File: source/redis_ssl_faq.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to encrypt your data in transit, it is recommended that the you download a certificate authority or certification authority from Rackspace to verify the authenticity of the certificate that Rackspace is using during the “handshake” process.  You can download the certificate authority here - http://ssl.rackspaceclouddb.com/rackspace-ca-2016.pem
+
+Within the ObjectRocket UI, you will also be provided with a key “fingerprint”.  This “fingerprint” will be located right next to the SSL connection string that you are going to use.  You can use this fingerprint to verify the encrypted connection during the “handshake” process.
```
Comment (dwjohn) [inline source/redis_ssl_faq.rst]: Remove this section - won't have "fingerprints" yet.

**Comment by SethSanchez** [comment]
@dwjohn How's it look after his changes?

File: source/redis_ssl_faq.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+You don’t need to do anything to “enable” SSL Encryption on their ObjectRocket Redis instance(s).  All new ObjectRocket Redis instances have SSL encryption enabled by default.  You will simply be presented with an extra set of connection strings that are SSL Encryption specific.  Currently, you are presented with both a Public and ServiceNet connection string.  With SSL Encryption, you will be presented with both an SSL Public and SSL ServiceNet connection string.
+
+What do I need to do with my application to use SSL Encryption
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You will need to use some type of SSL proxy in order to handle the SSL “handshake” that takes place between your client and the ObjectRocket Redis endpoint.  An example of this would be stunnel.  Once you have this in place, all that you need to do is point your application to the desired SSL encryption connection string.  There should be no additional code changes required on your end.  You may chose to do some additional verification of the server that you are connecting to via certificate “fingerprint” confirmation (see certificates and fingerprints below).
```
Comment (dwjohn) [inline source/redis_ssl_faq.rst]: Remove the last sentence about "fingerprint"

Comment (dwjohn) [inline source/redis_ssl_faq.rst]: Change you "will" to you "may"

**Comment by dwjohn** [comment]
+ 1 but want Andy to take one last look

**Comment by SethSanchez** [comment]
Derek confirmed merge in #main.

**Pull request merged**